CMakeLists.txt

```diff
@@ -246,7 +252,7 @@ foreach(PYTHON_SCRIPT ${TAICHI_AOT_PYTHON_LIST})
     string(REPLACE " " ";" PYTHON_SCRIPT ${PYTHON_SCRIPT})
     list(GET PYTHON_SCRIPT 0 PYTHON_FILE)
     list(GET PYTHON_SCRIPT 1 ARGS)
-    execute_process(COMMAND ${PYTHON_EXECUTABLE} ${PYTHON_FILE} ${ARGS})
+    execute_process(COMMAND ${PYTHON_EXECUTABLE} ${PYTHON_FILE} ${ARGS} --arch vulkan)
```

Comment:
@jim19930609  could you check this?

Reply:
oops actually this is included by mistake... gonna delete it in this PR